### PR TITLE
Ft docs init

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
#### Summary of Changes (What does this PR do?)
- Adds the docs project to the mono-repo
- Renders [this docs repo](https://github.com/airqo-platform/docs) obsolete (and should be deleted cc @Baalmart)

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Can't be tested manually
* However, follow the link in the deploy preview([this link](https://ft-docsinit-docs-preview-w7kzhvlewq-ew.a.run.app/)) to preview the changes
* You can also do static code analysis.

#### What are the relevant tickets?
- [AN-62](https://airqoteam.atlassian.net/browse/AN-62?atlOrigin=eyJpIjoiMTdmMzdlOTIwNmI3NGM0NWEzM2MyN2JlNWM2MWFkY2YiLCJwIjoiaiJ9)

#### Screenshots (optional)
image

### CODE REVIEW

File: k8s/netmanager/stage-airqo-platform-frontend.yaml [hunk 1]
```diff
@@ -65,7 +65,7 @@ spec:
     - protocol: TCP
       port: 80
       targetPort: 80
-      nodePort: 31104
+      nodePort: 31101
```
Comment (123MwanjeMike) [inline k8s/netmanager/stage-airqo-platform-frontend.yaml:68]: Switch old netmanager port for staging to the one reflected in the plan docs [here](https://docs.google.com/document/d/1r0wJLxr2jrtjk8oqnv-N5DBNcORYENeWQntDIjwhVLU/edit#heading=h.yyn93hqir91h)

File: k8s/docs/prod-airqo-docs.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
+###########################################################
```
Comment (123MwanjeMike) [inline k8s/docs/prod-airqo-docs.yaml:1]: [docs k8s manifests](https://github.com/airqo-platform/AirQo-frontend/pull/807/commits/e5d2be7268f27a48356e1b294b3a16367af1abe4)

File: docs/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,7 @@
+FROM node:16-alpine
```
Comment (123MwanjeMike) [inline docs/Dockerfile:1]: [add docker file](https://github.com/airqo-platform/AirQo-frontend/pull/807/commits/70b5de036faadd2ba1b2dc6a85adc69a53204882)

**Comment by 123MwanjeMike** [comment]
This PR depends on the backend PR below.
https://github.com/airqo-platform/AirQo-api/pull/1073
cc @NoahNsimbe  @Baalmart

**Review by Baalmart — APPROVED** [review]
LGTM, thanks @123MwanjeMike

**Pull request merged**